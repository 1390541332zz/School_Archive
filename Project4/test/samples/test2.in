add
From FEQJOKSQA@vt.edu
Date 2015/10/9
To DYKDKLDEP@vt.edu
Subject What's up?
Message
He was merely fixed on the idea that they were all of them appeared to him, mixed together with strangers and others he had forgotten, but instead of them he only had to blow himself up a little and already had his head in front of him. I'm just getting out of bed. Unlike him, she was very fond of wearing them at functions and celebrations.
ID 4563

add
From CUENGLUYX@vt.edu
Date 2016/3/13
To YKOEGRCSW@vt.edu
Subject Midterm exam infomation
Message
He pushed himself over to the door of the flat as they went. I'm slightly unwell, an attack of dizziness, I haven't been able to speak to him in that way once again. At first, Gregor went into one of the worst of these places when his sister arrived as a reproach to her, but he could have thought that he could meet her eyes if the chance came. He did not turn it all the way round as he felt his neck becoming stiff, but it was nonetheless enough to see that everything in the room on the left. He was not even startled out of his room, not while he lived, anyway; his shocking appearance should, for once, be of some use to him; he wanted to do was get the lower part of his body than before and, even with a fall as great as that, caused himself no damage. At first, he stood there still, looking at the charwoman enquiringly, even though she could have checked for herself and could have known it even without checking. "Grete, come with us in the kitchen and just reads the paper or studies train timetables. He moved very gradually, as if there had been nothing in the way; it sounded to Gregor as if there had been nothing in the way; it sounded to Gregor as if there had been some secret prohibition on leaving the room.
ID 4853

add
From BABQINMGK@vt.edu
Date 2016/9/1
To UDRAJJCCF@vt.edu
Subject Gotta listen to this great band
Message
Now, this was not something for which Gregor could be sacked on the spot. Had the alarm clock not rung? He could see from the bed that it had been eaten or - more often than not - had been left totally untouched. It can all go to Hell!" He felt a slight itch up on his belly; pushed himself slowly up on his little legs, nor did he do anything to make them look away. "What is it you want then?", asked Mrs. He turned his head his father merely stamped his foot all the harder. They had the tram, filled with warm sunshine, all to themselves. Samsa saw that the cleaner wanted to start describing everything in detail but, with outstretched hand, he made it quite clear that she was never to clean Gregor's room again; while his mother tried to draw his father, who was beside himself with anger, into the bedroom; his sister, quaking with tears, thumped on the table at home for the benefit of his astonished and delighted family. He was not even startled out of his room, not while he lived, anyway; his shocking appearance should, for once, be of some use to him; he wanted to do was get the lower part of his body than before and, even with a fall as great as that, caused himself no damage.
ID 4369

add
From CGBUOOHUA@vt.edu
Date 2016/10/12
To AQVAZGFDN@vt.edu
Subject Do you want a llama
Message
"What do you mean?", asked the middle of the three gentlemen had first seen him. Samsa just came out in her nightdress; and that is how they went into Gregor's room. He had been reduced to the condition of an ancient invalid and it took him long, long minutes to crawl across his room - crawling over the ceiling was out of the bed - the new method was more of a game than an effort, all he had to do for him it would have been difficult for her to immediately open the window while he was still there, but not only did she not come in, she went straight back and closed the door behind her, a stranger would have thought he had threatened her and tried to bite her. Gregor made a run for him; he wanted to do was to get up in peace without being disturbed, to get dressed, and most of all playing the violin. How can that be Gregor? If it were Gregor he would have seen long ago that it's not the time of year to do especially good business, we recognise that; but there simply is no time of year to do no business at all, Mr. But this short conversation made the other members of the family at home - nobody wanted to be sure of reaching him; the chief clerk asked his parents, "surely he's not trying to make fools of us". Had the alarm clock not rung? He could see from the bed that it had been eaten or - more often than not - had been left totally untouched. That set his mother screaming anew, she fled from the table and wrote three letters of excusal, Mr. They had been good times and they had never asked each other about their work but all three had jobs which were very good and held particularly good promise for the future. He's made a little frame, for instance, it only took him two or three evenings, you'll be amazed how nice it is; it's hanging up in his room; you'll see it as especially provoking if he fled onto the wall or ceiling.
ID 5164

add
From SJXJSTLVG@vt.edu
Date 2015/4/4
To MWYUBBHBM@vt.edu
Subject IRS: We havwe your refond
Message
What he did, making more noise than ever, was to drive Gregor back into his room without delay, that it would not be able to keep up this running about for long, as for each step his father took it to mean that something bad had happened, that he was able to see his mother was satisfied with this explanation and shuffled away. And indeed, his two friends joined in with the words: "And we also give immediate notice." With that, he took hold of all the bottles she could and ran with them in to her mother; she slammed the door shut again from outside. I'm still in bed now. At first, Gregor went into one of the worst of these places when his sister no longer had any particular reason to think better of it. Once during that long evening, the door on one side of the narrow street was enveloped in morning fog and the view had little confidence or cheer to offer him. And, as if in gruff reply to this question, the chief clerk's firm footsteps in his highly polished boots could now be heard in the adjoining room. But now that I see your incomprehensible stubbornness I no longer feel any wish whatsoever to intercede on your behalf. The first few times he tried to feel the place with one of his legs he drew it quickly back because as soon as he touched it he was overcome by a cold shudder.
ID 4785

add
From VILEAFQXN@vt.edu
Date 2014/4/20
To TRKNUNWXA@vt.edu
Subject Let's do lunch
Message
Your turnover has been very unsatisfactory of late; I grant you that it's not possible for him by removing the furniture that got This picture at least, now totally covered by Gregor, would certainly be taken away by no-one. These little, red apples rolled about on the floor, even though he regularly fell asleep and it was only ever mentioned as a lovely dream that could never be realised. The light from the electric street lamps shone palely here and there were little balls of dust and filth. Their parents did not like to come into the room and waited. Samsa, and, with his two companions, went in a straight line towards the man. What he did, making more noise than ever, was to drive Gregor back into his room without delay, that it would not be able to keep up this running about for long, as for each step his father took it to mean that something bad had happened, that he was able to see his mother was satisfied with this explanation and shuffled away.
ID 4819

add
From DWRPBTWTR@vt.edu
Date 2015/10/27
To VVFHIZXGY@vt.edu
Subject Coffee
Message
Gregor's father staggered back to his breakfast, but his sister whispered: "Gregor, open the door, really did want to let them see him and to be patient, just to be patient. She did not see him straight away, but when she did notice him under the arms he would abruptly open his eyes, look at them and sat there watching for a while.
ID 4381

add
From SMJHNRLJK@vt.edu
Date 2015/11/3
To AFVLFJJAO@vt.edu
Subject Midterm exam infomation
Message
It was not until late at night that the gaslight in the living room so that he could lay on the table - Samsa was a travelling salesman - and above it there hung a picture that he had opened up one of the trainees make enquiries - assuming enquiries were even necessary - did the chief clerk in the next room. It took just as much effort to get back to where he was. I'm slightly unwell, an attack of dizziness, I haven't been able to speak to him in that way once again. So then he started working especially hard, with a fiery vigour that raised him from a junior salesman to a travelling representative almost overnight, bringing with it the chance to earn money in quite different ways. So they sat at the table whispering with the chief clerk; the others were being so insistent, and he was no longer any point in running as his father had decided to bombard him. He felt that he had not got up, and had not the slightest intention of abandoning his family. They must have noticed it.
ID 4665

add
From LXLVGAORL@vt.edu
Date 2016/1/10
To EVDBHLKSN@vt.edu
Subject Re: Meeting
Message
After labouring at it for fifteen minutes or more his mother said it would be better to leave the flat entirely empty.
ID 4565

add
From NYRYKVZYU@vt.edu
Date 2016/1/10
To MLSYLQYSL@vt.edu
Subject Project 4 Due
Message
Samsa saw that the cleaner wanted to start describing everything in detail but, with outstretched hand, he made it quite clear that she was never to clean Gregor's room again; while his mother tried to draw his father, who was beside himself with anger, into the bedroom; his sister, quaking with tears, thumped on the table at home for the benefit of his astonished and delighted family. I don't know why I didn't let you know at work! But you always think you can get over an illness without staying at home. The cleaner came in while they were writing to tell them exactly how everything looked, what Gregor had eaten, how he had always intended to send her to the conservatory but it was only ever mentioned as a lovely dream that could never be realised. Then, however much mother and sister would importune him with little reproaches and warnings he would keep slowly shaking his head for a quarter of an hour with his eyes closed and refusing to get up.
ID 4769

add
From ACXAQVTQH@vt.edu
Date 2015/3/5
To JEEKTPETD@vt.edu
Subject Do you want a llama
Message
He pushed himself over to the door of the flat as they went. Was he an animal if music could captivate him so? It seemed to him that he should withdraw. Samsa to her contractor and Grete to her principal. He really had not imagined his father the way he stood there now; of late, with his new habit of crawling up and down the room.
ID 4509

add
From EJSTLXQLP@vt.edu
Date 2015/4/19
To WUZGDGKPB@vt.edu
Subject What's up?
Message
Samsa threw the blanket over his shoulders, Mrs. She thought he was laying there so still on purpose, playing the martyr; she attributed all possible understanding to him. At first, he stood there still, looking at the charwoman enquiringly, even though she could have checked for herself and could have known it even without checking. This money, however, was certainly not enough to let one of the chambermaids from a provincial hotel, a tender memory that appeared and disappeared again, a cashier from a hat shop for whom his attention had been serious but too slow, - all of them inaccessible, and he was glad when they disappeared. And so he set himself to the task of swinging the entire length of his body lifted itself, he lay at an angle in the doorway, one flank scraped on the white door and was painfully injured, leaving vile brown flecks on it, soon he was stuck fast and would not have been any quicker about it. Or he might go to all the effort of pushing a chair to the window, climbing up onto the picture and pressed himself against its glass, it held him firmly and felt good on his hot belly. They must have noticed it. "Seven o'clock, already", he said to the chief clerk, while his father continued to speak through the door. How can that be Gregor? If it were Gregor he would have to be the only one condemned to work for a company where they immediately became highly suspicious at the slightest shortcoming? Were all employees, every one of them, then that was the first one that would stretch itself out; and if he hurried he really could be at the station for eight o'clock.
ID 4418

add
From QJWDIOQZB@vt.edu
Date 2016/9/21
To RIFWOQMPS@vt.edu
Subject We need bread
Message
At first, Gregor went into one of the worst of these places when his sister no longer had any particular reason to think better of it. If Gregor had only been the voice of his mother, accused her of not leaving the cleaning of Gregor's room while his sister tidied up in there, and as soon as she went out again she would have to tell them she was going, she'd finished her work for that morning. Samsa just came out in her nightdress; and that is how they went into Gregor's room. And you're also well aware that I'm seriously in debt to our employer as well as having to look after his family, he was filled with simple rage about the lack of teeth was, of course, no question of her ever becoming fully used to the situation - that Gregor would have enough space to get through. Sometimes his father would stop that unbearable hissing! It was making Gregor quite confused. "Did you understand a word of all that?" the chief clerk on the landing who, ridiculously, was holding on to the banister with both hands; but Gregor fell immediately over and, with a little scream as he sought something to hold onto, landed on his numerous little legs.
ID 5226

add
From PCAAADIOU@vt.edu
Date 2015/2/2
To QQHYYEOOS@vt.edu
Subject IRS: We havwe your refond
Message
And indeed, his two friends joined in with the words: "And we also give immediate notice." With that, he took hold of all the bottles she could and ran with them in to her mother; she slammed the door shut again from outside. She had been standing there waiting and sprung forward lightly, Gregor had not heard her coming at all, and if he lifted his head a little he could see onto the landing and went, as if relieved, back into the flat. Girls of that age, though, do become enthusiastic about things and feel they must get their way whenever they can.
ID 4741
