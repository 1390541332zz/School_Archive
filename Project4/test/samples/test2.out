Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
Command: add
	Message added
